lib/htmlValidator.js

```diff
@@ -42,47 +42,51 @@ module.exports = function(app, callback) {
       params.htmlValidator.validator = 'http://localhost:' + (params.htmlValidator.port ? params.htmlValidator.port : '8888');
       options = params.htmlValidator;
     }
-    // spawn validator child process
-    promise = (() => {
-      return new Promise((resolve, reject) => {
-        validatorProcess = spawn(
-          'java', ['-Xss1024k', '-cp', vnu, 'nu.validator.servlet.Main', params.htmlValidator.port ? params.htmlValidator.port : '8888']
-        );
-
-        console.log('\u23F3  ' + ('Starting HTML validator...').bold.yellow);
-
-        validatorProcess.stdout.on('data', (data) => {
-          if (`${data}`.includes('Initialization complete')) {
-            clearTimeout(validatorTimeout);
-            console.log('✔️  ' + ('Validator initialization completed successfully. HTML validator listening on port: ' + params.htmlValidator.port).bold.green);
-            resolve();
-          }
-        });
 
-        validatorProcess.stderr.on('data', (data) => {
-          if (`${data}`.includes('No Java runtime present, requesting install.')) {
-            console.error('❌  ' + ('There was an error in authentication. Please make sure to have Java installed.').red);
-            reject();
-          }
-        });
+    // only run validator if in development mode
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
@kethinov checking for dev mode like this is working the way we want it to. Wrapping the promise and validatorProcess in this if check causes the validator to only run when the app is in dev mode. This is now what we wanted it to do, correct?